hwilib/devices/bitbox02.py

```diff
@@ -178,10 +181,14 @@ def enumerate(password: Optional[str] = None, expert: bool = False, chain: Chain
     Enumerate all BitBox02 devices. Bootloaders excluded.
     """
     result = []
-    for device_info in devices.get_any_bitbox02s():
+    devs = devices.get_any_bitbox02s()
+    if allow_emulators:
+        devs.append({'path': SIMULATOR_PATH.encode(), 'serial_number': '', 'product_string': 'bitbox02'})
+    for device_info in devs:
         path = device_info["path"].decode()
-        client = Bitbox02Client(path)
-        client.set_noise_config(SilentNoiseConfig())
+        client = Bitbox02Client(path=path)
+        if path != SIMULATOR_PATH:
+            client.set_noise_config(SilentNoiseConfig())
```

Comment:
`Bitbox02Client` constructor already assigns noiseconfig for both cases: `BitBoxNoiseConfig` for simulator and `CLINoiseConfig` for non-simulator. I personally wouldn't use ` client.set_noise_config(SilentNoiseConfig()`, but I wanted to leave it as it was already there.
Only difference between `BitBoxNoiseConfig` and `SilentNoiseConfig` is in `show_pairing` function, which is not called in `enumerate` function.
My suggestion here would be using `client.set_noise_config(SilentNoiseConfig())` or `client.set_noise_config(BitBoxNoiseConfig())` for both cases. I don't think setting in `init()` is suitable since constructor already initializes it.

Reply:
Then use the SilentNoiseConfig in both cases - don't change the config for non-simulators if that works, as it was put there for a reason (`Rationale: enumerate() should not show any dialogs.`).

Reply:
Okay, makes sense